extension/background.js

```diff
@@ -199,10 +199,12 @@ const createNotification = (title = '', message = '') => {
  */
 const createAccount = async (name, path) => {
```

Comment:
loading is not displaying after relaunch during create account if its first account